src/elements/review_by_id.jsx

```diff
@@ -17,19 +29,34 @@ function ReviewById() {
     getReviewsFromId(number).then((response) => {
       setReviewByIdState(response);
     });
-  }, []);
+  }, [pressed]);
   if (!reviewByIdState) {
     return <h2>loading...</h2>;
   } else {
     return (
-      <div className="reviewById">
-        <section>
-          <h2>{reviewByIdState.title}</h2>
-          <p>by</p>
-          <h2>{reviewByIdState.owner}</h2>
-        </section>
-        <img src={reviewByIdState.review_img_url} alt="shove here"></img>
-        <p className="reviewBody">{reviewByIdState.review_body}</p>
+      <div>
+        <div className="reviewById">
+          <section>
+            <div className="reviewTitle">
+              <h2>{reviewByIdState.title}</h2>
+              <p>by</p>
+              <h2>{reviewByIdState.owner}</h2>
+            </div>
+            <h5 className="voteText">Votes : {reviewByIdState.votes}</h5>
+            <button
+              className="vote"
+              disabled={pressed}
```

Comment:
the button is only disabled while we wait for the successful patch request, do we want users to be able to vote more than once? or do you want to disable the button for the user completely after it's clicked?